lib/ajv.js

```diff
@@ -413,11 +413,21 @@ function errorsText(errors, options) {
   options = options || {};
   var separator = options.separator === undefined ? ', ' : options.separator;
   var dataVar = options.dataVar === undefined ? 'data' : options.dataVar;
+  var extraInfo = options.extraInfo === true;
 
   var text = '';
+  var extraText = '';
   for (var i=0; i<errors.length; i++) {
     var e = errors[i];
-    if (e) text += dataVar + e.dataPath + ' ' + e.message + separator;
+    if (e) {
+      if(extraInfo) {
+        if (e.params && e.params.allowedValues)
+          extraText += ' (' + e.params.allowedValues.join(',') + ')';
+        else if (e.params && e.params.additionalProperty)
```

Comment:
there was a suggestion to log additional property previously, but given that it's data submitted from external source it is not secure. It is covered in https://github.com/epoberezkin/ajv/blob/master/FAQ.md#why-dont-additionalproperties-false-errors-display-the-property-name. I am going to close PR, sorry.